js/repl/ASTPanel.js

```diff
@@ -56,20 +56,22 @@ export default class ASTPanel extends React.Component<Props, State> {
     return null;
   }
 
-  onOptionSettingCheck(option: string) {
-    this.setState(prevState => ({
-      astOption: {
-        ...prevState.astOption,
-        [option]: !prevState.astOption[option],
-      },
-    }));
-
-    this.onChangeJson(option);
+  _onOptionSettingCheck(option: string) {
+    this.setState(
+      prevState => (
+        {
+          astOption: {
+            ...prevState.astOption,
+            [option]: !prevState.astOption[option],
+          },
+        },
+        this._onChangeJson(option)
```

Comment:
it's different with what I wanted to say.
There is no difference of count of `setState`.

I meant that you could get a return value from `_onChangeJson` and `setState` together.

Reply:
And there is also a chance this change is worse than the previous code. 
(If React has an optimization of reducing the number of `setState` call under the hood)

Reply:
@sujinleeme